helix-view/src/editor.rs

```diff
@@ -1222,7 +1305,7 @@ impl Editor {
                 .collect::<HashMap<_, _>>()
         });
 
-        if language_servers.is_empty() {
+        if language_servers.is_empty() && doc.language_servers.is_empty() {
```

Comment:
existing but that I noticed that I fixetd since its related (to the refreshing document language)